tests/test_cyto_utils/conftest.py

```diff
@@ -44,15 +44,15 @@ def fixture_metadata_input_file_s3() -> str:
     """
     Provide a metadata input file for cell_locations test data
     """
-    return "s3://cellpainting-gallery/test-cpg0016-jump/source_4/workspace/load_data_csv/2021_08_23_Batch12/BR00126114/test_BR00126114_load_data_with_illum.parquet"
+    return "s3://cellpainting-gallery/cpg0016-jump/source_4/workspace/load_data_csv/2021_08_23_Batch12/BR00126114/load_data_with_illum.parquet"
 
 
 @pytest.fixture(name="single_cell_input_file_s3")
 def fixture_single_cell_input_file_s3() -> str:
     """
     Provide a single cell input file for cell_locations test data
     """
-    return "s3://cellpainting-gallery/test-cpg0016-jump/source_4/workspace/backend/2021_08_23_Batch12/BR00126114/test_BR00126114.sqlite"
+    return "s3://cellpainting-gallery/cpg0016-jump/source_4/workspace/backend/2021_08_23_Batch12/BR00126114/BR00126114.sqlite"
```

Comment:
Erin's suggestion in https://github.com/broadinstitute/cellpainting-gallery/issues/80#issuecomment-1986375614 was to move away from JUMP since it is the most likely to change in the future - just checking if you've seen this advice.

If you're looking for an alternative file, I think this one sounds reasonable (and likely more stable)

`s3://cellpainting-gallery/cpg0017-rohban-pathways/broad/workspace/backend/2013_10_11_SIGMA2_Pilot/41744/41744.sqlite`

Reply:
(see https://cellpainting-gallery.s3.amazonaws.com/index.html#cpg0017-rohban-pathways/broad/workspace/backend/2013_10_11_SIGMA2_Pilot/41744/)

Reply:
Let's add this as a separate issue. I like doing a minimal set of changes here (same file new path) and then switching to a different data set later.

Reply:
Thanks @gwaybio and @kenibrewer, I stuck to the non-test data path as I wasn't certain what might occur and noticed the tests needed to be updated for other reasons. I feel it might be good to address this as Ken suggests in #375 .